Makefile

```diff
@@ -45,6 +45,8 @@ SHELL:=/bin/bash
 	    rm -f nx-X11/config/cf/nxversion.def; \
 	    rm -f bin/nxagent; \
 	    rm -f bin/nxproxy; \
+	    rm -f nx*/install-sh; \
```

Comment:
`mkinstalldirs` is tricky, we should be using `./install-sh -d` instead, since it's deprecated.